plugins/chat/app/jobs/regular/auto_join_channel_memberships.rb

```diff
@@ -2,7 +2,7 @@
 # frozen_string_literal: true
 
 module Jobs
-  class AutoManageChannelMemberships < ::Jobs::Base
+  class AutoJoinChannelMemberships < ::Jobs::Base
```

Comment:
Renamed this since `Manage` makes it sound like it adds _and_ removes and I didn't want any confusion